### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by manavluhar**
## Purpose of this pull request
 
This pull request (PR) is to improve README.md file

- Changed heading from **Magento Support Knowledge Base** to **Adobe Commerce Support Knowledge Base**.

- Added correct link for [LICENSE.txt](LICENSE.txt).

- Updated styling in **General Contribution Flow** section to have a correct view of the information.

- Applied other small changes related to works and style. 
## Affected Support KB pages
 

 
Not affecting any page on support.magento.com.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
+# Adobe Commerce Support Knowledge Base
+This repository contains the markdown source of articles, published in [Adobe Commerce Support Knowledge Base](https://support.magento.com/hc/en-us).
 
 ## Goals
 
-Our goal is to provide the Magento community with comprehensive and quality documentation on resolving Magento issues, Magento best practices, and sharing answers to questions our Support team typically gets. We believe to accomplish that goal, we need experts from the community to share their knowledge with us and each other. We are thankful to all of our contributors for improving Magento Support documentation.
+Our goal is to provide the Adobe Commerce Community with comprehensive and quality documentation on resolving Magento issues, Magento best practices, and sharing answers to questions our Support team typically gets. We believe that to accomplish that goal, We need experts from the community to share their knowledge with us and each other. We are thankful to all of our contributors for improving Adobe Commerce Support Knowledge Base documentation.
```
Comment (BarnyShergold) [inline README.md:6]:
```suggestion
Our goal is to provide the Adobe Commerce Community with comprehensive and quality documentation on resolving Magento issues, Magento best practices, and sharing answers to questions our Support team typically gets. We believe that to accomplish this goal, we need experts from the community to share their knowledge with us and each other. We are thankful to all of our contributors for improving Adobe Commerce Support Knowledge Base documentation.
```

File: README.md [hunk 2]
```diff
@@ -1,37 +1,36 @@
...
 
+### General Contribution Flow
 
-#### General contribution flow
-
-1. Fork this repo.
-1. Make edits on forked repo.
-1. Submit a Pull Request (PR) to this repo.
+1. Fork this Repository.
+1. Make edits on forked Repository.
```
Comment (BarnyShergold) [inline README.md:15]:
```suggestion
1. Make edits on the forked Repository.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,37 +1,36 @@
...
+   1. Knowledge Base (KB) writers reviews the PR within several days' time frame and adds label.
+   1. KB writer can approve/deny/request changes.
+   1. If approved, KB writer adds labels corresponding to the level of input provided in PR and internal subject matter expert (SME) reviews the PR.
+   1. SME can approve/deny/request changes.
+1. Once all corrections are done (if any requested) and both the KB writer and SME approve the PR, The KB writer imports content to the internal repo and merges it internally.
+1. The  **magento/knowledge-base** repository synchronizes with the internal one in 20 minutes.
+1. Once the repositories are synced, your PR gets closed and you get reward points.
+
+
+- For more details on contribution flow, Please refer to the [Contributor's Guide](.github/CONTRIBUTING.md) of Knowledge Base.
```
Comment (BarnyShergold) [inline README.md:31]: You have to stick to one list style

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (manavluhar) [reply README.md]: Thank you  @BarnyShergold, I committed the changes.

**Review by almarchenko — CHANGES_REQUESTED** [review]
Could you please get rid of hard tabs mentioned in the failed Linting test.
Thank you

**Comment by manavluhar** [comment]
Thank you @almarchenko, I updated the PR.

File: README.md [hunk 4]
```diff
@@ -1,37 +1,34 @@
...
-### Licensing
-
-This project is licensed under the OSL 3.0 License. See [LICENSE](LICENSE) for more information.
+### General Contribution Flow
+
+- Fork this Repository.
+- Make edits on the forked Repository.
+- Submit a Pull Request (PR) to this Repository.
+- Tests are run:
+  - Adobe CLA - making sure the Adobe Open Source Contributor License Agreement is signed.
```
Comment (BarnyShergold) [inline README.md:18]: Linting is giving an md0007 error - if this repo is set up as devdocs then the 2nd level ordered list needs to be indented by 3 spaces not 2 - will need to be done for all 2nd level list items below

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (manavluhar) [reply README.md:18]: Thank you **Barny Shergold**, I tried giving 3 spaces and also 2 spaces. still, it's throwing MD0007 issue. Can you please suggest the changes?

Comment (BarnyShergold) [reply README.md:18]: Just been testing this locally - try indenting the second level by FOUR spaces. I did this and locally MDL returned no errors

Comment (manavluhar) [reply README.md]: BOOM! it worked Thank you so much Barny Shergold

**Review by BarnyShergold — APPROVED** [review]
Hurrah!

**Review by almarchenko — APPROVED** [review]

**Comment by almarchenko** [comment]
@magento import code to magento-kb/knowledge-base

**Comment by magento-engcom-team** [comment]
@almarchenko the branch with code successfully imported into`magento-kb/knowledge-base` repository. Branch name: `imported-magento-knowledge-base-48`.

**Pull request merged**